source/framework/core/inc/TRestEventProcess.h

```diff
@@ -229,4 +259,9 @@ class TRestEventProcess : public TRestMetadata {
 
     ClassDefOverride(TRestEventProcess, 3);  // Base class for a REST process
 };
+
+#define _ApplyCut(evt)        \
```

Comment:
Personally, I am not fan of macros. Also, I cannot see where this macro is used.

Reply:
It could used in for example in `TRestRawSignalAnalysisProcess`, to replace the cut code. 

`if (ApplyCut()) return nullptr; return fSignalEvent;` --> `_ApplyCut(fSignalEvent)`

I think it is more clear to use this macro.